src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        Map<String, Integer> statistics = readData(fromFileName);
+        writeData(toFileName, statistics);
+    }
+
+    private Map<String, Integer> readData(String fileName) {
```

Comment:
You can use `java.nio.file.Files` to read all lines at once and then just return an array of String instead of using map. For example like this:

![image](https://github.com/user-attachments/assets/083f9ce5-8393-4931-9ff5-a61155b068e1)